commerce_coordinator/apps/frontend_app_ecommerce/views.py

```diff
@@ -101,18 +107,30 @@ def get(self, request):
 
         try:
             order_data = OrderHistoryRequested.run_filter(request, params)
+            orders = chain.from_iterable(
+                order_set["results"] for order_set in order_data
+            )
+            orders = sorted(
+                orders,
+                key=lambda order: date_conv(order["date_placed"]),
+                reverse=True,
+            )
 
-            output_orders = []
-
-            for order_set in order_data:
-                output_orders.extend(order_set['results'])
+            if params["cutoff_in_days"]:
```

Comment:
Why do we need this check since we are not passing this parameter in URL query params and instead setting it explicitly in a dict?